static/src/js/__tests__/App.test.js

```diff
@@ -87,9 +87,14 @@ jest.mock('../components/Layout/Layout', () => {
   })
 })
 
+jest.mock('../utils/getPermittedUser', () => ({
```

Comment:
I don't think you'll need this if you use App Config version

Reply:
Removed, thanks!